SentryTestUtils/ClearTestState.swift

```diff
@@ -5,6 +5,13 @@ public func clearTestState() {
     TestCleanup.clearTestState()
 }
 
+public func resetUserDefaults() {
+    if let appDomain = Bundle.main.bundleIdentifier {
+        UserDefaults.standard.removePersistentDomain(forName: appDomain)
+        UserDefaults.standard.synchronize()
```

Comment:
That's correct. I'm aware of that, but it leads to flaky tests when not calling it. I forgot to add a comment. Does it make sense now?

Reply:
Interesting, I made some tests while creating an App, and it did not make any difference calling `synchronize` since it did not save user default synchronous.

All good.